### ISSUE TITLE
refinery-core 0.5.1 version constraints not compatible with rusqlite 0.25

### ISSUE THREAD

**Issue by paulwib** [rust-db/refinery#163]
Cargo.toml:

```
[package]
name = "refinery-issue"
version = "0.1.0"
edition = "2018"

[dependencies]
rusqlite = { version = "0.25" }
refinery = { version = "0.5", features = ["rusqlite"] }
```

Output:

```
$ cargo check
    Updating crates.io index
error: failed to select a version for `libsqlite3-sys`.
    ... required by package `rusqlite v0.25.0`
    ... which is depended on by `refinery-issue v0.1.0 (refinery-issue)`
versions that meet the requirements `^0.22.0` are: 0.22.2, 0.22.1, 0.22.0

the package `libsqlite3-sys` links to the native library `sqlite3`, but it conflicts with a previous package which links to `sqlite3` as well:
package `libsqlite3-sys v0.18.0`
    ... which is depended on by `rusqlite v0.23.0`
    ... which is depended on by `refinery-core v0.5.0`
    ... which is depended on by `refinery v0.5.0`
    ... which is depended on by `refinery-issue v0.1.0 (refinery-issue)`

failed to select a version for `libsqlite3-sys` which could resolve this conflict
```

I also tried adding an exact version of `refinery-core`:

```
[dependencies]
rusqlite = { version = "0.25" }
refinery = { version = "0.5", features = ["rusqlite"] }
refinery-core = { version = "0.5.1"}
```

Which gave maybe a more informative error - the version constraint doesn't look right:

```
$ cargo check
    Updating crates.io index
error: failed to select a version for `rusqlite`.
    ... required by package `refinery-core v0.5.1`
    ... which is depended on by `refinery v0.5.0`
    ... which is depended on by `refinery-issue v0.1.0 (refinery-issue)`
versions that meet the requirements `>=0.23, <0.25` are: 0.24.2, 0.24.1, 0.24.0, 0.23.1, 0.23.0

the package `rusqlite` links to the native library `sqlite3`, but it conflicts with a previous package which links to `sqlite3` as well:
package `rusqlite v0.25.0`
    ... which is depended on by `refinery-issue v0.1.0 (refinery-issue)`

failed to select a version for `rusqlite` which could resolve this conflict
``` 

I'm a bit confused by this as I see #159 intended to support 0.25, but the `Cargo.toml` crate source seems to have a different version constraint from the original source: https://docs.rs/crate/refinery-core/0.5.1/source/Cargo.toml.

**Comment by jxs** [comment]
Hi, yeah I haven't yet released a version with rusqlite 0.25. I should probably do it

**Comment by gbaranski** [comment]
> Hi, yeah I haven't yet released a version with rusqlite 0.25. I should probably do it

hi, are there any updates on this?

**Comment by jxs** [comment]
version 0.6 released! see https://github.com/rust-db/refinery/pull/165
thanks
